config/dev.exs

```diff
@@ -36,8 +36,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :skillswheel, Skillswheel.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "postgres",
-  password: "postgres",
+  username: "jackcarlisle",
+  password: "",
```

Comment:
To make this more generic, we could instead get future developers to create a postgres user instead of having to edit this line and gitignore it.

You should be able to add the user `postgres` by running:

```
/usr/local/Cellar/postgresql/9.5.4_1/bin/createuser -s postgres
```

Or whatever postgres version you are using, let me know if that is something we should look to do and if so, if that command fixes the problem